packages/blade/src/components/Icons/HomeIcon/HomeIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import { Svg, Path } from '../_Svg';
+import type { IconComponent } from '..';
+import useIconProps from '../useIconProps';
+
+const HomeIcon: IconComponent = ({ size, color }) => {
+  const { height, width, iconColor } = useIconProps({ size, color });
+
+  return (
+    <Svg height={height} width={width} viewBox="0 0 24 24" fill="none">
+      <Path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
```

Comment:
I think these should be `fillRule`, `clipRule` (TS should ideally be warning these). Similarly at other places.

Reply:
This resolved the issue

Reply:
Created https://github.com/razorpay/blade/issues/648 to track this